src/Graphics/Color.cs

```diff
@@ -14,942 +14,1121 @@
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Numerics;
+using System.Runtime.InteropServices;
 
-namespace MoonWorks.Graphics
+namespace MoonWorks.Graphics;
+
+/// <summary>
+/// Describes a 32-bit packed color.
+/// </summary>
+[StructLayout(LayoutKind.Sequential, Size = 4, Pack = 4)]
```

Comment:
This really isn't necessary, it's just 4 bytes so default alignment will mean the structure is just 4 bytes.